src/java.base/share/man/java.md

```diff
@@ -3073,7 +3073,9 @@ The following items describe the syntax of `java` argument files:
 -   The argument file size must not exceed MAXINT (2,147,483,647) bytes.
 
 -   The launcher doesn't expand wildcards that are present within an argument
-    file.
+    file. That means, an asterisk  `*` is passed on as-is to the starting VM.
+    For example `*.java` stays `*.java` and is not expanded to `Foo.java`,
+    `Bar.java`, etc. like on some command line shell.
 
```

Comment:
This expansion now seems to be in conflict with the later statement:
```
Wildcards (*) aren't allowed in these lists (such as specifying *.java).
```
?? To be honest I'm not sure what that later statement is actually trying to say.

Reply:
Never mind I see now this was already flagged.

Reply:
I think that sentence was trying to convey that the "wildcard literal `*` will be treated literally and the launcher will not attempt to expand it". I think this sentence can be removed altogether, given that this semantic is already explained a few sentences above by the change in this PR.